js_modules/dagit/packages/core/src/runs/RunTableNew.tsx

```diff
@@ -291,25 +325,55 @@ const RunRow: React.FC<{
               </Link>
             </Box>
           )}
-          <RunTags
-            tags={
-              [
-                run.pipelineSnapshotId
-                  ? {
-                      key: DagsterTag.SnapshotID,
-                      value: run.pipelineSnapshotId?.slice(0, 8) || '',
-                      link: run.pipelineSnapshotId
-                        ? getPipelineSnapshotLink(run.pipelineName, run.pipelineSnapshotId)
-                        : undefined,
-                    }
-                  : null,
-                run.tags.find((tag) => tag.key === DagsterTag.Partition),
-              ].filter((x) => x) as TagType[]
-            }
-            mode={isJob ? (run.mode !== 'default' ? run.mode : null) : run.mode}
-            onAddTag={onAddTag}
-          />
+          <Box flex={{direction: 'row', gap: 8, wrap: 'wrap'}}>
+            <Box>
+              <BaseTag
+                fillColor={Colors.Gray100}
+                label={
+                  <ButtonLink
+                    onClick={() => {
+                      setShowRunTags(true);
+                    }}
+                  >
+                    {run.tags.length} tag{run.tags.length === 1 ? '' : 's'}
+                  </ButtonLink>
+                }
+              />
+            </Box>
+            <RunTagsWrapper>
+              <RunTags
+                tags={
+                  [
+                    run.pipelineSnapshotId
+                      ? {
+                          key: DagsterTag.SnapshotID,
+                          value: run.pipelineSnapshotId?.slice(0, 8) || '',
+                          link: run.pipelineSnapshotId
+                            ? getPipelineSnapshotLink(run.pipelineName, run.pipelineSnapshotId)
+                            : undefined,
+                        }
+                      : null,
+                    run.tags.find((tag) => tag.key === DagsterTag.Partition),
+                  ].filter((x) => x) as TagType[]
+                }
+                mode={isJob ? (run.mode !== 'default' ? run.mode : null) : run.mode}
+                onAddTag={onAddTag}
+              />
+            </RunTagsWrapper>
+          </Box>
         </Box>
+        {showRunTags || showRunTagsIcon ? (
+          <ShortcutHandler
+            key="runtabletags"
+            onShortcut={() => {
+              setShowRunTags((showRunTags) => !showRunTags);
+            }}
+            shortcutLabel="t"
+            shortcutFilter={(e) => e.code === 'KeyT'}
+          >
+            {null}
+          </ShortcutHandler>
+        ) : null}
```

Comment:
What does this end up looking like?

Reply:
Oh it does render the "t" if you hold command